releasenotes/notes/os-capacity-deploy-hook-b52e87c0819df6fd.yaml

```diff
@@ -0,0 +1,17 @@
+---
+features:
+  - |
+    Automatic deployment for OpenStack Capacity via a Kayobe service
+    deploy hook using kolla admin credentials.
+issues:
```

Comment:
Issues is for known issues/bugs. Isn't the described issue more of a feature?

Reply:
It was a bit generic but I assumed it might be helpful if someone does not have kolla credentials configured properly in admin-openrc.sh or if those credentials don't have permissions to access all projects.

Can remove it though depending on if that information is in the right place or not.

Reply:
Is it a bug that could/should be fixed?